terraform/common/groups.tf

```diff
@@ -269,12 +286,46 @@ resource "aws_iam_policy" "parameter_store" {
   policy = data.aws_iam_policy_document.parameter_store.json
 }
 
-
 resource "aws_iam_role_policy_attachment" "secreteditor_role_policies" {
   role       = aws_iam_role.secreteditor.name
   policy_arn = aws_iam_policy.parameter_store.arn
 }
 
+# Deployers group, deployments role - no MFA required
+
+resource "aws_iam_role" "deployments" {
+  name               = "Deployments"
+  assume_role_policy = data.aws_iam_policy_document.assume_role_no_mfa.json
```

Comment:
This should be the policy not the document

Reply:
The other four references to `assume_role_policy` all directly consume the json, e.g.

```
resource "aws_iam_role" "secreteditor" {
  name               = "SecretEditor"
  assume_role_policy = data.aws_iam_policy_document.assume_role_if_mfa_present.json
}
```

Reply:
ok 👍